specification/maps/data-plane/Search/preview/2021-11-01-preview/geocoding.json

```diff
@@ -0,0 +1,686 @@
+            "successfulRequests": {
+              "description": "Number of successful requests in the batch",
+              "type": "integer",
+              "readOnly": true
+            },
+            "totalRequests": {
+              "description": "Total number of requests in the batch",
+              "type": "integer",
+              "readOnly": true
+            }
+          }
+        },
+        "batchItems": {
+          "description": "Array containing the batch results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GeocodingBatchResponseItem"
+          }
+        }
+      }
+    },
+    "Features": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/FeaturesItem"
+      }
+    },
+    "FeaturesItem": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "description": "must be Feature",
+          "enum": [
+            "Feature"
+          ]
+        },
+        "id": {
+          "type": "string",
+          "description": "ID for feature returned"
+        },
+        "properties": {
+          "type": "object",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "One of: \n* Address\n* RoadBlock\n* RoadIntersection",
+              "readOnly": true
+            },
+            "confidence": {
+              "$ref": "#/definitions/Confidence"
+            },
+            "matchCodes": {
+              "$ref": "#/definitions/MatchCodes"
+            },
+            "address": {
+              "$ref": "#/definitions/Address"
+            },
+            "geocodePoints": {
+              "$ref": "#/definitions/GeocodePoints"
+            }
+          }
+        },
+        "geometry": {
+          "$ref": "#/definitions/GeoJSONPoint"
+        },
+        "bbox": {
+          "$ref": "#/definitions/Bbox"
+        }
+      }
+    },
+    "GeocodingResponse": {
+      "description": "This object is returned from a successful Geocoding call",
+      "type": "object",
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/FeatureType"
+        },
+        "features": {
+          "$ref": "#/definitions/Features"
+        },
+        "nextLink": {
+          "$ref": "#/definitions/NextLink"
+        }
+      }
+    },
+    "Address": {
+      "description": "The address of the result",
+      "type": "object",
+      "properties": {
+        "addressLine": {
+          "description": "AddressLine that includes Street Name and Number",
+          "type": "string",
+          "readOnly": true
+        },
+        "locality": {
+          "description": "locality property",
+          "type": "string",
+          "readOnly": true
+        },
+        "neighborhood": {
+          "description": "neighborhood property",
+          "type": "string",
+          "readOnly": true
+        },
+        "adminDistricts": {
```

Comment:
Should we add a description like Bing Maps? Below:

The subdivision name in the country or region for an address. This element is typically treated as the first order administrative subdivision, but in some cases it is the second, third, or fourth order subdivision in a country, dependency, or region.

Reply:
this is an array field. we provide the information in its sub-field.

Reply:
added